history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
+-  rendering setting fixes for copy and paste actions and floating point
+   rendering
+-  image interpolation can now be turned on and off in the clients
+-  more readable timestamps
+-  measurement table fixes
+-  directories are now preserved when downloading multiple original files
+-  various improvements to the imageJ plugin's handling of ROIs and
```

Comment:
ImageJ:
 * Improve handling of ROI
also for rendering
I will break it into 2 entries to make it clearer considering the increasing number of floating point data.
Fix floating point rendering issues
Copy paste settings.